asciidoc-confluence-publisher-client/src/main/java/org/sahli/asciidoc/confluence/publisher/client/http/ConfluenceRestClient.java

```diff
@@ -162,6 +163,16 @@ public void addAttachment(String contentId, String attachmentFileName, InputStre
         });
     }
 
+    @Override
+    public void addAttachment(String contentId, String attachmentFileName, Path attachmentContent) {
+        HttpPost addAttachmentRequest = this.httpRequestFactory.addAttachmentRequest(contentId, attachmentFileName, attachmentContent);
+        sendRequestAndFailIfNot20x(addAttachmentRequest, (response) -> {
+            //closeInputStream(attachmentContent);
+            System.out.println("failed");
```

Comment:
Printing “failed” to stdout doesn't seem to be the best possible way to handle this.

Suggestion:
- Use a logger if that is also done in the rest of the code base, or otherwise print to stderr if that is also done elsewhere.
- In either case, at least improve the message to make it more specific, so people reading the message will understand what is the problem/situation.